src/PlatformCompatibilityTests/ServiceControlCompatibilityTests/SampleTests.cs

```diff
@@ -1,26 +1,50 @@
 ﻿namespace ServiceControlCompatibilityTests
 {
     using NUnit.Framework;
+    using System;
+    using System.Linq;
 
     [TestFixture]
     class SampleTests : SqlScTest
     {
         [Test]
-        public void SampleTestA()
+        public async void SampleTestA()
         {
-            var details = ServiceControl.Get<ServiceControlDetails>("");
```

Comment:
Yeah this was copy-pasted from the SC tests with the intent that we would build something more use case specific here. The SC tests have access to all of the internal objects used for the API so the generic approach is easy there.